jest.config.js

```diff
@@ -1,5 +1,6 @@
 module.exports = {
   preset: 'ts-jest/presets/js-with-babel',
+  testPathIgnorePatterns: ['RbGen*'],
```

Comment:
why is this needed ? Tests are never run with generated components

Reply:
Yes, you are right, I added that because when the script failed in the middle, it didn't clean up the generated components, I discovered later that normally they are removed.

I will remove that.